classes/capture.py

```diff
@@ -73,30 +73,14 @@ def captureFrame(self, frame):
 		# write the frame to video file
 		self.out.write(frame)
 
-		self.timestamp = int(round(time.time()))
+		self.timestamp = self.get_time()
 
+	# return time form file or take from device
+	# The biggest problem is that the device don't have internet connection
+	# and because of that the time is incorrect!
+	def get_time(self):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/saoron_3361/Cardigan/file/2695937705/issues/source?bid=3323288&fileBranchId=3340798#l81)